src/CoherenceBuild/CoherenceBuild.csproj

```diff
@@ -0,0 +1,15 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp1.0</TargetFramework>
+    <OutputType>Exe</OutputType>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Extensions.CommandLineUtils" Version="1.0.1" />
+    <PackageReference Include="NuGet.Packaging" Version="3.5.0" />
```

Comment:
I'm working on removing the dependency in a follow up - https://github.com/aspnet/Coherence/issues/170.